### PR TITLE
fix(obs-backgroundremoval-git): fix install for pacstall 3

### PR DESCRIPTION
**Pull request by Zahrun**

### CODE REVIEW

File: packages/obs-backgroundremoval-git/obs-backgroundremoval-git.pacscript [hunk 1]
```diff
@@ -37,6 +37,7 @@ build() {
 }
 
 install() {
+  cd build || exit
```
Comment (Henryws) [inline packages/obs-backgroundremoval-git/obs-backgroundremoval-git.pacscript:40]:
```suggestion
  cd build
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/obs-backgroundremoval-git/obs-backgroundremoval-git.pacscript (hunk 1 above)
Comment (Zahrun) [reply packages/obs-backgroundremoval-git/obs-backgroundremoval-git.pacscript:40]: image

Comment (Henryws) [reply packages/obs-backgroundremoval-git/obs-backgroundremoval-git.pacscript:40]: I am aware. Functions run in an environment where any non-zero exit code fails out completely. So saying `cd build || exit` is redundant.

Comment (Zahrun) [reply packages/obs-backgroundremoval-git/obs-backgroundremoval-git.pacscript]: As you wish. I now learned the `git commit --no-verify` option to by-pass the pre-commit hooks.

**Comment by Henryws** [comment]
CI fails because it can’t `cd build`

**Comment by Zahrun** [comment]
> CI fails because it can’t `cd build`

I think pacstall 3.0.1 changed the behavior. Can close this.

**Comment by Henryws** [comment]
> 

How? The only change is in unguarded variables.

**Comment by Zahrun** [comment]
> 

In 3.0.0, when I cd to build directory in build(), install() would start on parent directory. In 3.0.1, install() starts in same directory left by build()

**Comment by Henryws** [comment]
You’re saying it would reset the dir on every function?

**Comment by Zahrun** [comment]
> You’re saying it would reset the dir on every function?

That’s the behavior I observed in 3.0.0

**Pull request closed without merging**